desktop-style.css

```diff
@@ -0,0 +1,252 @@
+body {
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  height: 3504.94px;
+  margin: 0;
+}
+
+.mobile-nav { display: none; }
+
+header {
+  height: 636px;
+  background: url("assets/images/Header-llustration-desktop.svg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: top 0 left 27px;
+  padding: 0;
+}
+
+.wrapper {
+  background:
+    url("assets/images/right-about-me.svg"),
+    url("assets/images/left-about-me.svg"),
+    url("assets/images/illustration-1-contact-form-desktop.svg"),
+    url("assets/images/illustration-2-contact-form-desktop.svg"),
+    url("assets/images/illustratoin-3-contact-form-desktop.svg");
+  background-repeat: no-repeat;
+  background-position:
+    top 1900px right 0,
+    top 2150px left -50px,
+    bottom 72.49px left 0,
+    bottom 156.14px left 274px,
+    bottom 74.83px right 0;
+}
+
+.navigation {
+  display: flex;
+  justify-content: center;
+}
+
+.desktop-nav {
+  list-style: none;
+  display: flex;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
+  font-weight: 500;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  text-align: center;
+  align-items: center;
+  color: #344563;
+}
+
+.desktop-nav li { padding: 10px 12px; }
+
+.headline {
+  display: flex;
+  width: 65%;
+  margin: auto;
+  padding: 160px 0;
+  height: 436px;
+}
+
+.headline h1 { height: 114px; }
+
+.headline p {
+  height: 96px;
+  margin: 24px 0 0;
+}
+
+.socials { margin-top: 24px; }
+
+.arrow {
+  display: flex;
+  height: 114px;
+  justify-content: center;
+  align-items: flex-end;
+  margin-top: 24px;
+}
+
+.bar { display: none; }
+
+.recent-works {
+  height: 1084px;
+  width: 90%;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  padding: 80px 0;
+}
+
+.title,
+.title h2 {
+  display: block;
+  width: 447px;
+  height: 48px;
+  margin: auto;
+}
+
+.recent-works-cards {
+  height: 988px;
+  width: 100%;
+  margin: 52px auto 0;
+  display: grid;
+  grid-template-columns: auto auto auto;
+  row-gap: 32px;
+  column-gap: 24px;
```

Comment:
- [ ] The width of your buttons is not properly aligned with the original design. Kindly ensure you stick to the design guidelines(margins, padding, width, etc) as much as possible.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/154493919-d43c7cf6-6994-4150-97cc-f02a7208cb54.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/154493841-0b8699ad-024b-4e72-8c1b-8616a540b459.png)

Reply:
@AnselemOdims I don't seem to have this error on my end. I just checked and the button size is consistent.

I attached a video for you to see it in action 

https://user-images.githubusercontent.com/55185309/154504470-467445ef-d1de-4172-8f1c-47692264623d.mp4